src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,804 @@
++++
+title = "User Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 15
++++
+
+## Overview
+
+This guide is intended for users who have a Submariner environment set up and want to verify the installation and learn more about how to
+use Submariner and the main capabilities it provides. This guide assumes that there are two Kubernetes clusters,
+**cluster2** and **cluster3**, forming a cluster set, and that the Broker is deployed into a separate cluster **cluster1**.
+
+{{% notice tip %}}
+Make sure you have `subctl` [set up](../deployment/subctl/#installation). Regardless of how Submariner was deployed, `subctl` can be used
+for various verification and troubleshooting tasks, as shown in this guide.
+{{% /notice %}}
+
+{{% notice note %}}
+This guide focuses on a non-Globalnet Submariner deployment.
+{{% /notice %}}
+
+### 1. Validate the Installation
+
+#### On the Broker
+
+The Broker facilitates the exchange of metadata information between the connected clusters, enabling them to discover one another.
+The Broker consists of a set of Custom Resource Definitions (CRDs) only; there are no Pods or Services deployed with it.
+
+This command validates that the Broker namespace has been created in the Broker cluster:
+
```

Comment:
Since multiple clusters are running, you need to choose which cluster `kubectl` talks to. `subctl` will just follow. Perhaps this is something that we need to document as part of https://submariner.io/operations/deployment/subctl/?

Reply:
I think you anyways need to export KUNECONFIG for `kubectl config use-context cluster1` to work.

Reply:
Done